texar/evals/bleu_tool.py

```diff
@@ -0,0 +1,217 @@
+# Copyright 2019 The Texar Authors. All Rights Reserved.
```

Comment:
I meant to name the functions as `bleu_tool`. Now that we have other names for the functions, here let's name the file as `bleu_transformer.py`